src/address-manager/index.js

```diff
@@ -27,7 +29,7 @@ class AddressManager {
   /**
    * Get peer listen multiaddrs.
    *
-   * @returns {Array<Multiaddr>}
+   * @returns {Array<multiaddr>}
```

Comment:
Nit: Using upper case first char in type names is an idiomatic pattern in TS & think it would be best to follow it. Assuming we do not want to rename everywhere you could define a type at the end of the file e.g.:

```js
/**
 * @typedef {import('multiaddr')} Multiaddr
 */

Which is also how we do it in js-ipfs. I think it would be also less confusing as in js-ipfs we use `Multiaddr`.

Reply:
Yes, I mentioned this topic in the main post of the PR. 
I would also prefer to use `Multiaddr`, but the drawback is doing the `@typedef` import in every single file that uses them, which are a lot. I don't have a strong opinion on this, and I am happy to change if we think that is a better solution.

Reply:
changed to `Multiaddr` everywhere